modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotDeliveryWorker.java

```diff
@@ -425,7 +425,8 @@ public void deleteSlot(Slot slot) {
         if (null != slotsOfQueue) {
             slotsOfQueue.remove(slot.getId());
         } else {
-            log.warn("Slot has been deleted by the SlotDeliveryWorker before the SlotDeletionExecutor can get to it. Slot : " + slot);
+            log.debug("Slot has been deleted by the SlotDeliveryWorker before the SlotDeletionExecutor can get to it." +
```

Comment:
When a subscriber leaves the node this scenario can happen. Hence this should be a debug log.